CHANGELOG.md

```diff
@@ -39,6 +39,12 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* Upgrade Rosetta to v0.7.2 [#560](https://github.com/provenance-io/provenance/issues/560)
```

Comment:
Need to copy the `## [v1.7.5]...` heading and put it above with appropriate changes.

```suggestion
## [v1.7.6](https://github.com/provenance-io/provenance/releases/tag/v1.7.6) - 2021-12-14

### Improvements

* Upgrade Rosetta to v0.7.2 [#560](https://github.com/provenance-io/provenance/issues/560)
```